packages/travix-ui-kit/components/stepper/stepper.scss

```diff
@@ -0,0 +1,101 @@
+.ui-stepper {
+  display: flex;
+  height: 39px;
+  justify-content: space-between;
+
+  .ui-stepper-button.ui-stepper-button_increase,
+  .ui-stepper-button.ui-stepper-button_decrease {
```

Comment:
Do these selectors need to be nested? Also, if we are following BEM, they should be renamed to `.ui-stepper__button`, `ui-stepper__button--increase` and `ui-stepper__button--decrease`.

Reply:
I've used correct BEM `container__element-modificator`